src/adagrams.js

```diff
@@ -1,15 +1,141 @@
+const letterPoolObj = {
+  A: 9, 
+  B: 2, 
+  C: 2, 
+  D: 4, 
+  E: 12, 
+  F: 2, 
+  G: 3, 
+  H: 2, 
+  I: 9, 
+  J: 1, 
+  K: 1, 
+  L: 4, 
+  M: 2, 
+  N: 6, 
+  O: 8, 
+  P: 2, 
+  Q: 1, 
+  R: 6, 
+  S: 4, 
+  T: 6, 
+  U: 4, 
+  V: 2, 
+  W: 2, 
+  X: 1, 
+  Y: 2, 
+  Z: 1
+}
+
 export const drawLetters = () => {
   // Implement this method for wave 1
+  let letterList = [];
+  let letterPoolCopy = letterPoolObj;
+
+  for (let i = letterList.length; i < 10; ++i) {
+    let objKeys = Object.keys(letterPoolObj);
+    let randLetter = objKeys[Math.floor(Math.random() * objKeys.length)];
```

Comment:
Since objKeys is an array of 26 letters, grabbing a random letter from objKeys doesn't reflect the distribution of letters as described by letterPoolObj. You'd want to create an array that contains 9 As, 2 Bs, 2 Cs ... and then grab your letter from this array.